certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
-                                        "description": "method name",
-                                        "title": "routingKey",
-                                        "x-ms-visibility": "internal"
-                                    }
-                                },
-                                "required": [
-                                    "routingKey"
-                                ]
-                            },
-                            "combinedRoute": {
-                                "type": "string",
-                                "description": "created by policy",
-                                "title": "combinedRoute",
-                                "x-ms-visibility": "internal"
-                            }
-                        },
-                        "required": [
-                            "id",
-                            "enrichment"
-                        ]
-                    },
-                    "description": "elements"
-                }
-            }
-        }
-    },
-    "parameters": {},
-    "responses": {},
-    "securityDefinitions": {
-        "api_key": {
-            "type": "apiKey",
-            "in": "header",
-            "name": "x-api-key"
+        {
+          "propertyName": "Categories",
+          "propertyValue": "Business Management; Data"
         }
-    },
-    "security": [],
-    "tags": []
+      ],
+  "consumes": [
+      "application/json"
+  ],
+  "produces": [
+      "application/json"
+  ],
+  "paths": {
+      "/api/v1/projects": {
+          "get": {
+              "summary": "Get All Projects",
+              "operationId": "GetAllProjects",
+              "x-ms-visibility": "internal",
+              "parameters": [
+                  {
+                      "name": "with_contracts",
+                      "type": "boolean",
+                      "in": "query",
+                      "default": true,
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines that Contract ID must be included in the response.",
+                      "description": "Determines that Contract ID must be included in the response.",
+                      "required": true
+                  },
+                  {
+                      "name": "module_type",
+                      "type": "string",
+                      "in": "query",
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "description": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "required": true
+                  }
+              ],
+              "description": "Returns a list of user's projects",
+              "responses": {
+                  "200": {
+                      "description": "default",
```

Comment:
@boiv may I ask you please change the description from "default" to something like "OK" or "Success". Please fix all responses in the swagger file.

Reply:
@Amjed-Ayoub 
Updated `description` with value `OK` in all responses